### PR TITLE
New instrument cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYIV Instrument Cal CSVs [DT-158]](https://trello.com/c/I1InRCiO/158-pyiv-instrument-cal-csvs-dt-158)

File: calibration/DOSTAD/CGINS-DOSTAD-00227__20220426.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+227,CC_conc_coef,"[-2.677333E+00, 1.210953E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_227_Calibration_Certificate_2022-04-26.pdf"
+227,CC_csv,"[3.11358E-03, 1.30733E-04, 2.50351E-06, 2.32461E+02, -3.46572E-01, -5.80057E+01, 4.53733E+00]",
```
Comment (IDCalTech) [inline calibration/DOSTAD/CGINS-DOSTAD-00227__20220426.csv:3]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00466__20220426.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+466,CC_conc_coef,"[-1.722816E+00, 1.124639E+00]","[offset, slope]; source_file DOSTA-D_Optode-4831_SN_466_Calibration_Certificate_2022-04-26.pdf"
+466,CC_csv,"[2.85179E-03, 1.21990E-04, 2.37240E-06, 2.32010E+02, -3.09932E-01, -5.43607E+01, 4.56618E+00]",
```
Comment (IDCalTech) [inline calibration/DOSTAD/CGINS-DOSTAD-00466__20220426.csv:3]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01103__20220427.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
...272.64, 273.44, 274.25, 275.05, 275.85, 276.66, 277.46, 278.26, 279.07, 279.87, 280.67, 281.48, 282.28, 283.08, 283.89, 284.69, 285.49, 286.3, 287.1, 287.91, 288.71, 289.51, 290.32, 291.12, 291.92, 292.73, 293.53, 294.33, 295.14, 295.94, 296.74, 297.55, 298.35, 299.15, 299.96, 300.76, 301.56, 302.37, 303.17, 303.97, 304.78, 305.58, 306.38, 307.19, 307.99, 308.79, 309.59, 310.4, 311.2, 312, 312.81, 313.61, 314.41, 315.21, 316.02, 316.82, 317.62, 318.42, 319.22, 320.03, 320.83, 321.63, 322.43, 323.23, 324.04, 324.84, 325.64, 326.44, 327.24, 328.04, 328.84, 329.64, 330.45, 331.25, 332.05, 332.85, 333.65, 334.45, 335.25, 336.05, 336.85, 337.65, 338.45, 339.25, 340.05, 340.85, 341.65, 342.45, 343.25, 344.04, 344.84, 345.64, 346.44, 347.24, 348.04, 348.84, 349.63, 350.43, 351.23, 352.03, 352.82, 353.62, 354.42, 355.22, 356.01, 356.81, 357.61, 358.4, 359.2, 360, 360.79, 361.59, 362.38, 363.18, 363.97, 364.77, 365.56, 366.36, 367.15, 367.95, 368.74, 369.53, 370.33, 371.12, 371.92, 372.71, 373.5, 374.29, 375.09, 375.88, 376.67, 377.46, 378.26, 379.05, 379.84, 380.63, 381.42, 382.21, 383, 383.79, 384.58, 385.37, 386.16, 386.95, 387.74, 388.53, 389.32, 390.11, 390.9, 391.68, 392.47, 393.26]",
```
Comment (IDCalTech) [inline calibration/NUTNRB/CGINS-NUTNRB-01103__20220427.csv:8]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01062__20220427.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
...272.74, 273.55, 274.37, 275.18, 275.99, 276.81, 277.62, 278.43, 279.25, 280.06, 280.87, 281.69, 282.5, 283.31, 284.13, 284.94, 285.75, 286.57, 287.38, 288.2, 289.01, 289.82, 290.64, 291.45, 292.26, 293.08, 293.89, 294.7, 295.52, 296.33, 297.14, 297.96, 298.77, 299.58, 300.4, 301.21, 302.02, 302.84, 303.65, 304.46, 305.28, 306.09, 306.9, 307.71, 308.53, 309.34, 310.15, 310.97, 311.78, 312.59, 313.4, 314.22, 315.03, 315.84, 316.65, 317.46, 318.28, 319.09, 319.9, 320.71, 321.52, 322.34, 323.15, 323.96, 324.77, 325.58, 326.39, 327.2, 328.02, 328.83, 329.64, 330.45, 331.26, 332.07, 332.88, 333.69, 334.5, 335.31, 336.12, 336.93, 337.74, 338.55, 339.36, 340.17, 340.98, 341.79, 342.6, 343.41, 344.21, 345.02, 345.83, 346.64, 347.45, 348.26, 349.06, 349.87, 350.68, 351.49, 352.3, 353.1, 353.91, 354.72, 355.52, 356.33, 357.14, 357.94, 358.75, 359.55, 360.36, 361.17, 361.97, 362.78, 363.58, 364.39, 365.19, 366, 366.8, 367.6, 368.41, 369.21, 370.02, 370.82, 371.62, 372.42, 373.23, 374.03, 374.83, 375.63, 376.44, 377.24, 378.04, 378.84, 379.64, 380.44, 381.24, 382.04, 382.84, 383.64, 384.44, 385.24, 386.04, 386.84, 387.64, 388.44, 389.24, 390.04, 390.83, 391.63, 392.43, 393.23, 394.02, 394.82]",
```
Comment (IDCalTech) [inline calibration/NUTNRB/CGINS-NUTNRB-01062__20220427.csv:8]: Looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20220427.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
...61, 272.42, 273.23, 274.05, 274.86, 275.67, 276.49, 277.3, 278.11, 278.93, 279.74, 280.55, 281.37, 282.18, 282.99, 283.81, 284.62, 285.43, 286.24, 287.06, 287.87, 288.68, 289.5, 290.31, 291.12, 291.93, 292.75, 293.56, 294.37, 295.18, 295.99, 296.81, 297.62, 298.43, 299.24, 300.05, 300.87, 301.68, 302.49, 303.3, 304.11, 304.92, 305.74, 306.55, 307.36, 308.17, 308.98, 309.79, 310.6, 311.41, 312.22, 313.03, 313.84, 314.65, 315.46, 316.27, 317.08, 317.89, 318.7, 319.51, 320.32, 321.13, 321.94, 322.75, 323.56, 324.37, 325.18, 325.98, 326.79, 327.6, 328.41, 329.22, 330.02, 330.83, 331.64, 332.45, 333.25, 334.06, 334.87, 335.68, 336.48, 337.29, 338.09, 338.9, 339.71, 340.51, 341.32, 342.12, 342.93, 343.73, 344.54, 345.34, 346.15, 346.95, 347.76, 348.56, 349.37, 350.17, 350.97, 351.78, 352.58, 353.38, 354.18, 354.99, 355.79, 356.59, 357.39, 358.2, 359, 359.8, 360.6, 361.4, 362.2, 363, 363.8, 364.6, 365.4, 366.2, 367, 367.8, 368.6, 369.4, 370.19, 370.99, 371.79, 372.59, 373.39, 374.18, 374.98, 375.78, 376.57, 377.37, 378.16, 378.96, 379.76, 380.55, 381.35, 382.14, 382.93, 383.73, 384.52, 385.32, 386.11, 386.9, 387.69, 388.49, 389.28, 390.07, 390.86, 391.65, 392.45, 393.24, 394.03, 394.82]",
```
Comment (IDCalTech) [inline calibration/NUTNRB/CGINS-NUTNRB-01107__20220427.csv:8]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20220223.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+ACS-256,CC_tbins,"[0.849639, 1.255952, 2.456774, 3.467083, 4.473529, 5.502083, 6.513200, 7.496000, 8.483200, 9.491250, 10.503600, 11.516957, 12.508750, 13.480455, 14.481200, 15.505200, 16.494000, 17.492692, 18.496000, 19.496800, 20.503846, 21.486400, 22.497600, 23.504167, 24.502222, 25.496000, 26.495000, 27.504444, 28.485200, 29.499600, 30.496923, 31.500000, 32.511481, 33.488077, 34.547419]",
+ACS-256,CC_taarray,SheetRef:CC_taarray,
+ACS-256,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (IDCalTech) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20220223.csv:9]: Looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00227__20220426.csv (hunk 1 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00227__20220426.csv:3]: looks good

File: calibration/DOSTAD/CGINS-DOSTAD-00466__20220426.csv (hunk 2 above)
Comment (cdobs) [inline calibration/DOSTAD/CGINS-DOSTAD-00466__20220426.csv:3]: looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01062__20220427.csv (hunk 4 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01062__20220427.csv:8]: spot check - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01103__20220427.csv (hunk 3 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01103__20220427.csv:8]: spot check - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20220427.csv (hunk 5 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01107__20220427.csv:8]: spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20220223.csv (hunk 6 above)
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20220223.csv:9]: spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20220223__CC_taarray.ext [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+-0.010734,-0.007167,-0.005052,-0.004339,-0.003911,-0.003511,-0.002874,-0.002375,-0.001922,-0.001397,-0.00093,-0.000575,-0.000188,0.000084,0.000356,0.000692,0.000814,0.001,0.001114,0.001134,0.001085,0.000978,0.000808,0.000375,0.000293,0,-0.000244,-0.000385,-0.000579,-0.000897,-0.001047,-0.001204,-0.001367,-0.001535,-0.001785
+-0.011456,-0.007787,-0.00561,-0.004878,-0.004415,-0.004012,-0.003369,-0.002916,-0.00242,-0.001802,-0.00142,-0.001041,-0.000539,-0.000217,-0.000042,0.000318,0.000508,0.000673,0.000913,0.00091,0.000898,0.000872,0.000689,0.000372,0.00025,0,-0.00026,-0.000391,-0.00057,-0.00082,-0.001007,-0.00118,-0.001307,-0.001459,-0.001776
+-0.012146,-0.008474,-0.006214,-0.005521,-0.004978,-0.004617,-0.003947,-0.003482,-0.002931,-0.00229,-0.001946,-0.001469,-0.001034,-0.000722,-0.000388,-0.000072,0.00014,0.000409,0.000637,0.000637,0.000723,0.00071,0.000574,0.000286,0.000253,0,-0.000204,-0.000325,-0.000464,-0.000763,-0.000887,-0.001071,-0.001179,-0.001389,-0.001626
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20220223__CC_taarray.ext:82]: spot check - looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20220223__CC_tcarray.ext [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+-0.011742,-0.00997,-0.00928,-0.008738,-0.008466,-0.008188,-0.007966,-0.007482,-0.007071,-0.006828,-0.006427,-0.006035,-0.005614,-0.005202,-0.004902,-0.004387,-0.003958,-0.00338,-0.002908,-0.002369,-0.001996,-0.001361,-0.001003,-0.000668,-0.00029,0,0.000289,0.000605,0.00078,0.000944,0.000924,0.001105,0.001333,0.0012,0.001255
+-0.011135,-0.009349,-0.008412,-0.008159,-0.007962,-0.007734,-0.007408,-0.007092,-0.006559,-0.006279,-0.005748,-0.005492,-0.00499,-0.004739,-0.004275,-0.003856,-0.003385,-0.002881,-0.002605,-0.002058,-0.001811,-0.001326,-0.000927,-0.000674,-0.000164,0,0.00021,0.000357,0.000583,0.000908,0.000911,0.000967,0.001014,0.000926,0.000827
+-0.010498,-0.008999,-0.008173,-0.007723,-0.007456,-0.007279,-0.006936,-0.006712,-0.00623,-0.005995,-0.005492,-0.005174,-0.004845,-0.004494,-0.004106,-0.003702,-0.003328,-0.002843,-0.002424,-0.001951,-0.001625,-0.000988,-0.000889,-0.000657,-0.000144,0,0.000169,0.000391,0.000505,0.000687,0.00068,0.000813,0.000885,0.00091,0.000894
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20220223__CC_tcarray.ext:82]: spot check - looks good

**Review by cdobs — APPROVED** [review]
Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20220223__CC_taarray.ext (hunk 7 above)
Comment (IDCalTech) [reply calibration/OPTAAD/CGINS-OPTAAD-00256__20220223__CC_taarray.ext:82]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20220223__CC_tcarray.ext (hunk 8 above)
Comment (IDCalTech) [reply calibration/OPTAAD/CGINS-OPTAAD-00256__20220223__CC_tcarray.ext:82]: Looks good

**Review by IDCalTech — APPROVED** [review]
Looks good, ready to merge

**Pull request merged**